src/widgets/unit.jsx

```diff
@@ -21,7 +21,6 @@ var MathOutput   = require("../components/math-output.jsx");
 var { SignificantFigures, displaySigFigs } = require("../sigfigs.jsx");
 
 var ALL = "all";
-var SOME = "some";
```

Comment:
That one was used in the editor but not the widget itself if I remember correctly.